EdFi.Buzz.UI.Angular/src/app/Features/SurveyAnalytics2/surveyAnalytics2.component.html

```diff
@@ -36,6 +36,11 @@ <h1 *ngIf="!showSearchResults">Click "Search" to Show Results</h1>
 <h1 *ngIf="showSearchResults">Search Results</h1>
 
 <div *ngIf="showSearchResults" class="row" >
+  <div *ngIf="surveyMetadataList.length === 0" class="col-12 col-md-6">
```

Comment:
Do remove col-md-6 so that it extends the full length of the container.

Reply:
Done